### ISSUE TITLE
[Feature] Template Badge: allow the display of a badge

### ISSUE THREAD

**Issue by Nickduino** [piitaya/lovelace-mushroom#1552]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Is your feature request related to a problem?

No

### Describe the solution you'd like

I know the semantics aren't easy here since I'd like to put a badge on the icon of a badge :-D

A picture being worth a thousand words, here it is:

![image](https://github.com/user-attachments/assets/8c4a0b89-078f-46eb-94e2-8f7ee6f467e9)

### Additional context

This badge pops up on my dashboard when there is a weather alert in my area. In this case it's yellow (so not too serious) and due to the **wind** (hence the wind icon as a badge of the icon of the badge if you're following).

**Comment by Nickduino** [comment]
While you're at is, making these badges accept numeric values (just like with https://github.com/piitaya/lovelace-mushroom/issues/1551)

**Comment by piitaya** [comment]
Badges are designed to be minimalist element. Unfortunatly, It's not implemented because it's too small. Cards must be used if you want to display more information.

**Comment by Nickduino** [comment]
@piitaya Paul, in light of https://github.com/piitaya/lovelace-mushroom/pull/1755, would you reconsider placing a badge on the icon of a badge?

**Comment by piitaya** [comment]
Badge on a badge? 😅 IMO, it's too small. In the tile card and template card, it already very small. I have to fight with the Home Assistant UX team to keep it 😂 So I would not prefer to add it on the template badge.

**Comment by Nickduino** [comment]
> IMO, it's too small

I made a mock-up. Of course the decision is yours (I just ask you to keep in mind users might find something useful, even when you don't see the use case for your own dashboard) but I think it's useable on my 1080p 13" laptop. In this example, I'm not displaying some major info but I know how close the thunderstorm is (3 km).

<img width="196" height="121" alt="Image" src="https://github.com/user-attachments/assets/2329ae51-30ef-49fb-aad0-be17f2b67dfc" />

On the left, the red badge is the same size as on the Template card. On the right, I reduced the circle by 20% but I kept the font the same size.
